types/database/schema/ChannelList.types.ts

```diff
@@ -54,6 +56,7 @@ export interface ChatSchema {
   status: string;
   isMe: boolean;
   isContinuous: boolean;
+  getById: (db: SQLiteDatabase, id: string) => Promise<ChatSchema>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new method `getById` has been added to the `ChatSchema` interface. This method takes a `db` of type `SQLiteDatabase` and an `id` of type `string` as parameters and returns a Promise that resolves with a `ChatSchema`. Make sure that all classes implementing this interface also implement this method.

```diff
+ getById: (db: SQLiteDatabase, id: string) => Promise<ChatSchema>;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->